djvm/src/main/kotlin/net/corda/djvm/costing/RuntimeCost.kt

```diff
@@ -28,12 +64,19 @@ class RuntimeCost(
     /**
      * Increment the accumulated cost by a long integer.
      */
-    fun increment(incrementBy: Long) = incrementAndCheck(UnaryOperator { value ->
-        Math.addExact(value, incrementBy)
-    })
+    fun increment(incrementBy: Long) = incrementAndCheck(incrementBy)
 
     /**
      * Increment the accumulated cost by one.
      */
     fun increment() = increment(1L)
+
+    private companion object {
+        /**
+         * A set of threads to which cost accounting will be disabled.
+         */
+        private val filteredThreads: List<Thread> = emptyList()
```

Comment:
How are the `filteredThreads` set?

Reply:
No idea! I think it's just another example of Tommy over-engineering this.